src/intellij/scala.ipr.SAMPLE

```diff
@@ -157,6 +157,10 @@
   <component name="ProjectCodeStyleConfiguration">
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
     <code_scheme name="Project" version="173">
+      <ScalaCodeStyleSettings>
```

Comment:
No, I don't know how this differs from `<codeStyleSettings language="Scala">`. Maybe it's an XML thing.